wiki.md

```diff
@@ -0,0 +1,176 @@
+# AltaPay WooCommerce Plugin
+
+AltaPay, headquartered in Denmark, is an internationally focused fintech company within payments with the mission to make payments less complicated. We help our merchants grow and expand their business across payment channels by offering a fully integrated seamless omni-channel experience for online, mobile and instore payments, creating transparency and reducing the need for manual tasks with one centralized payment platform.
+
+AltaPay’s platform automizes, simplifies, and protects the transaction flow for shop owners and global retail and e-commerce companies, supporting and integrating smoothly into the major ERP systems. AltaPay performs as a Payment Service Provider operating under The Payment Card Industry Data Security Standard (PCI DSS).
+
+WooCommerce is an open-source eCommerce platform, built on WordPress. You can modify and customize it to meet your (merchant's) needs, and works alongside other plugins.
+
+# WooCommerce Payment plugin installation guide
+
+Installing this plug-in will enable the web shop to handle card transactions through AltaPay's gateway.
+
+
+**Table of Contents**
+
+[Prerequisites](#prerequisites)
+
+[Installation](#installation)
+
+[Configuration](#configuration)
+
+[Troubleshooting](#troubleshooting)
+
+
+
+# Prerequisites
+
+Minimum system requirements are:
+- WordPress min. 5.0 – max. 5.8.2
+- WooCommerce min. 3.9.0 - max. 5.9.0
+- PHP 7.0+
+- PHP-bcmath library installed.
+- PHP-curl MUST be enabled.
```

Comment:
In order to be consistent, shouldn't this be part of the `Supported Versions` section like we did in Magento2?

Reply:
added  under Supported versions section

Reply:
@maf-embraceit is the table of contents updated with it?

Reply:
@emicha I've updated the table of contents.